lib/views/screens/home_screen.dart

```diff
@@ -0,0 +1,232 @@
+import 'package:angeleno_project/controllers/overlay_provider.dart';
+import 'package:angeleno_project/views/screens/advanced_security_screen.dart';
+import 'package:angeleno_project/views/screens/password_screen.dart';
+import 'package:angeleno_project/views/screens/profile_screen.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../controllers/auth0_user_api_implementation.dart';
+import '../../controllers/user_provider.dart';
+import '../../models/user.dart';
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({super.key});
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  final GlobalKey<ScaffoldState> scaffoldKey = GlobalKey<ScaffoldState>();
+  final Auth0UserApi auth0UserApi = Auth0UserApi();
+  late UserProvider userProvider;
+  late User user;
+  late OverlayProvider overlayProvider;
+  int _selectedIndex = 0;
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  Future<void> _unsavedDataDialog(final int futureIndex) async =>
+    showDialog<void>(
+      context: context,
+      barrierDismissible: false,
+      builder: (final BuildContext context) => AlertDialog(
+        title: const Text('You have unsaved changes'),
+        content: const SingleChildScrollView(
+          child: ListBody(
+            children: <Widget>[
+              Text('Your changes have not been saved. Discard changes?')
+            ],
+          ),
+        ),
+        actions: <Widget>[
+          TextButton(
+            child: const Text('Cancel'),
+            onPressed: () {
+              Navigator.of(context).pop();
+            },
+          ),
+          TextButton(
+            child: const Text('Ok'),
+            onPressed: () {
+              Navigator.of(context).pop();
+              userProvider.toggleEditing();
+              _navigationSelected(futureIndex);
+            },
+          ),
+        ],
+      ),
+    );
+
+  Future<void> _navigationSelected(final int index) async {
+
+    if (userProvider.isEditing && index != 0) {
+      _unsavedDataDialog(index);
+    } else {
+      // Could use a cleaner implementation
+      if ([3, 4, 5, 6].contains(index)) {
+        switch(index) {
+          case 3:
+            await launchUrl(
+              Uri.parse('https://angeleno.lacity.org/')
+            );
+            break;
+          case 4:
+            await launchUrl(
+              Uri.parse('https://angeleno.lacity.org/apps')
+            );
+            break;
+          case 5:
+            await launchUrl(
+              Uri.parse('https://angeleno.lacity.org/help')
```

Comment:
Assuming these URLs will be updated when we have our new Angeleno website. Maybe we can have it as a variable instead.

Reply:
I don't think it's worth abstracting that out. When we make the changes I think we can just make the changes via a branch, but let me know your thoughts.